core/lib/dal/src/blocks_dal.rs

```diff
@@ -1228,16 +1240,16 @@ impl BlocksDal<'_, '_> {
         &mut self,
         raw_batches: Vec<StorageL1Batch>,
     ) -> anyhow::Result<Vec<L1BatchWithMetadata>> {
-        let mut l1_batches = Vec::with_capacity(raw_batches.len());
+        let mut l1_batches_with_metadata = Vec::with_capacity(raw_batches.len());
         for raw_batch in raw_batches {
             let block = self
                 .map_storage_l1_batch(raw_batch)
                 .await
-                .context("get_l1_batch_with_metadata()")?
+                .context("map_storage_l1_batch()")?
                 .context("Block should be complete")?;
-            l1_batches.push(block);
+            l1_batches_with_metadata.push(block);
```

Comment:
Nit / slightly off-topic: "block" should be renamed to "batch" in the local var naming and in `context()` message.